pkgs/by-name/to/tofu-ls/package.nix

```
@@ -23,6 +23,12 @@ buildGoModule (finalAttrs: {
     "-s"
   ];
 
+  # Generate schema
+  # https://github.com/opentofu/tofu-ls/issues/129#issuecomment-3457512823
+  preBuild = ''
+    go generate .
```

Should be:
```suggestion
    go generate ./...
```

But it isn't going to work because it wants network access.